Sluggo-SwiftUI/Views/TicketEditDetail.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  TicketEditDetail.swift
+//  Sluggo-SwiftUI
+//
+//  Created by Troy Ebert on 2/4/23.
+//
+
+import Foundation
+import SwiftUI
+
+struct TicketEditDetail: View {
+    
+    @EnvironmentObject var identity: AppIdentity
+    @StateObject var alertContext = AlertContext()
```

Comment:
If alertContext is used, it needs to be attached to the view with a call to:
`.alert(context: alertContext)`
otherwise present errors won't do anything

Reply:
I'll do that